src/app/action.cpp

```diff
@@ -137,15 +200,38 @@ void Action::set(QString key)
                 this->gpio.watcher.addPath(this->gpio.value.fileName());
             }
             else {
-                qDebug() << "[Dash][Action]" << this->key_ << ": failed to open" << active_low; // temp
+                QString debugStr;
+                QDebug stream(&debugStr);
+                stream << "[Action] " << this->key_ << ": failed to open" << active_low; // temp
+                DASH_LOG(info) << debugStr.toStdString();
             }
         }
         else {
-            qDebug() << "[Dash][Action]" << this->key_ << ": failed to open" << this->gpio.value.fileName(); // temp
+            QString debugStr;
+            QDebug stream(&debugStr);
+            stream << "[Action]" << this->key_ << ": failed to open" << this->gpio.value.fileName(); // temp
+            DASH_LOG(info) << debugStr.toStdString();
         }
     }
     else if (!this->key_.isNull()) {
-        qDebug() << "[Dash][Action]" << this->key_ << ": setting action as key"; // temp
-        this->shortcut.setKey(QKeySequence::fromString(this->key_));
+        QString debugStr;
+        QDebug stream(&debugStr);
+        stream << "[Action]" << this->key_ << ": setting action as key"; // temp
+        DASH_LOG(info) << debugStr.toStdString();
+        // if + is present, it means sequence is multiple keys.
+        // I'd have implemented this by looking at the .count() of the QKeySequence
+        // but it doesn't consider modifiers (ctrl, shift, etc) to be a discrete key
+        if(!this->key_.contains('+')) 
```

Comment:
@rsjudka see here for the new split of single key vs multikey

Reply:
yeah i figured thats what this was

does this break anything if someone were to use the '+' key?

Reply:
Oh that's a good point... I can test this soon, but the worst case would be + sign falling back to QShortcut (only triggered state, no press/release events).

Reply:
Looks like it detects "+" as "Shift++" so should be good.